source/content.css

```diff
@@ -122,9 +122,15 @@
 .toolbar-group:last-child {
 	display: inline-block;
 }
-.toolbar-group .toolbar-item:not(.js-saved-reply-container):not(.rgh-upload-btn) {
+./* Hide Specific toolbar items*/
+.toolbar-group .toolbar-item[data-prefix="@"], .toolbar-item[data-prefix="#"], .toolbar-group:first-child, .toolbar-item[data-prefix="`"] {
 	display: none;
-}
+	}
+/*Add Some Spacing*/
```

Comment:
All comments:
- only the first letter should be capitalized
- spaces after `/*` and before `*/`

Reply:
Capitalization still missing

Reply:
What should it be?
I dont know how to do it but it would be nice to add to the below 

![image](https://user-images.githubusercontent.com/16872793/33950953-abc3ee64-dffb-11e7-9de7-7a51baa6102b.png)